src/redecomp/RedecompMesh.hpp

```diff
@@ -148,9 +150,14 @@ class RedecompMesh : public mfem::Mesh
    * 
    * @param partitioner Method of partitioning the elements
    * @param n_parts Number of parts to partition the mesh into
+   * @param ghost_length Length around the domain border to include ghost elements
```

Comment:
Minor, but it is really more of a "fuzz" width beyond the current domain border. Length around the domain border makes it sound like you only include ghost elements on a portion of the domain border.